packages/shared/src/hooks/useTagAndSource.ts

```diff
@@ -199,7 +199,7 @@ export default function useTagAndSource({
     ],
   );
 
-  const onFollowSource = useCallback(
+  const onUnblockSource = useCallback(
```

Comment:
Why did we end up changing the definitions of these functions, I think it contributed a lot to refactors we could've avoided. It is great that we do refactors, though I think for experiments, we should keep it to minimum since things might still change after the result.

Reply:
the problem is that the new event is literally called "follow" and would conflict with this one which is an "unblock". Beyond the experiment it might be better to clarify what the method does by giving it a consistent name. What do you think?

Reply:
I agree, we should have these updated, but I think now is not yet the best time as this would still be running in an experiment. I can't exactly remember how things look now if the process still fits the narrative of these functions, but still, I think we should make the refactor at the end of the experiments.